### DISCUSSION TITLE
3주차 과제 질문

### DISCUSSION THREAD

**Discussion by ghost** [HeoJiye/Github-Study#18] [질문 게시판]
혹시 제 레포지터리에 짝꿍 교환일기가 없으면 3주차 과제를 수행하기 어려운가요?

**Comment by HeoJiye** [comment]
이 질문을 이제 봤는데 상대방의 레포를 fork한 이후 상대방의 레포에서 수정 사항이 있을 경우 sync로 동기화로 가져와보는 것, 내 레포든 상대방의 레포를 fork한 레포이든 pull과 patch를 사용해보는 게 과제의 목적이에요. 

순서가 어떻게 되든 한번씩 경험해봤다면 상관없습니다!
